content/security/docs/network.md

```diff
@@ -338,6 +338,119 @@ kubectl apply -f cluster-issuer.yaml
+
+To get this identity from Istio, the `istio-agent` sends a request known as a certificate signing request (or CSR) to the Istio control plane. This CSR contains the service account token so that the workload's identity can be verified before being processed. This verification process is handled by `istiod`, which acts as both the Registration Authority (or RA) and the CA. The RA serves as a gatekeeper that makes sure only verified CSR makes it through to the CA. Once the CSR is verified, it will be forwarded to the CA which will then issue a certificate containing a [SPIFFE](https://spiffe.io/) identity with the service account. This certificate is called a SPIFFE verifiable identity document (or SVID). The SVID is assigned to the requesting service for identification purposes and to encrypt the traffic in transit between the communicating services.
+
+![Default flow for Istio Certificate Signing Requests](./images/default-istio-csr-flow.png)
+
+#### How Certificate Signing Works in Istio with ACM Private CA
+You can use a cert-manager add-on called the Istio Certificate Signing Request agent ([istio-csr](https://cert-manager.io/docs/projects/istio-csr/)) to integrate Istio with ACM Private CA. This agent allows Istio workloads and control plane components to be secured with cert manager issuers, in this case ACM Private CA. The *istio-csr* agent exposes the same service that *istiod* serves in the default config of validating incoming CSRs. Except, after verification, it will convert the requests into resources that cert manager supports (i.e. integrations with external CA issuers). 
+
+Whenever there's a CSR from a workload, it will be forwarded to *istio-csr*, which will request certificates from ACM Private CA. This communication between *istio-csr* and ACM Private CA is enabled by the [AWS Private CA issuer plugin](https://github.com/cert-manager/aws-privateca-issuer). Cert manager uses this plugin to request TLS certificates from ACM Private CA. The issuer plugin will communicate with the ACM Private CA service to request a signed certificate for the workload. Once the certificate has been signed, it will be returned to *istio-csr*, which will read the signed request, and return it to the workload that initiated the CSR.
+
+![Flow for Istio Certificate Signing Requests with istio-csr](./images/istio-csr-with-acm-private-ca.png)
+
+
+#### Setup Instructions
+1. Start by following the same [setup instructions](https://aws.github.io/aws-eks-best-practices/security/docs/network/#setup-instructions) outlined above to complete the following:
```

Comment:
Just an FYI, not required for you to do here: Instead of numbering each step "1.", "2.", "3." and so on, you can number each step "1."  As long as you indent any sub-bullets or code blocks three spaces, each step will number sequentially. In the future, if you ever need to insert a step in the middle, you don't have to go through and renumber everything.

Reply:
Sure thing. Thanks @chrisnegus